ringcentral/http/client.py

```diff
@@ -17,7 +17,11 @@ def send(self, request):
         response = None
 
         try:
-            response = self.load_response(request.prepare())
+            prepared = request.prepare()
+            # Ref: https://github.com/requests/requests/issues/1736#issuecomment-28470217
+            if hasattr(request, 'multipart_mixed') and request.multipart_mixed:
+                prepared.headers['Content-Type'] = prepared.headers['Content-Type'].replace('multipart/form-data;', 'multipart/mixed;')
```

Comment:
Do not bring this into client! Do this in builder!!! Instead of setting a flag on builder.